pages/sight.vue

```diff
@@ -182,20 +156,31 @@
             Values
           </h1>
           <p>
-            Our values are empathy, solidarity, teamwork, commitment and, of
-            course, pride in being IEEE SIGHT.
+            We hope to create and implement sustainable solutions to make a
+            difference in the lives of people.
           </p></v-col
         >
       </v-row>
 
-      <spacer gap="3" />
+      <spacer gap="5" />
       <v-row class="text-center"
         ><v-col cols="12">
-          <h1 style="font-family: 'Hammersmith One', sans-serif !important">
-            About Us
-          </h1>
-          <v-card>
-            <v-card-text style="padding: 1rem">
+          <v-card elevation="11" outlined shaped tile>
+            <v-card color="primary">
+              <v-card-text
+                style="padding: 1rem"
+                class="text-center white--text headline"
+              >
+                About Us
```

Comment:
Try to make everything consistent!
Instead of this type of About us(heading), you can opt for "Get ivolved" type of heading.
Try to make this change in steering commitee also.